src/System.Data.Odbc/src/Common/System/Data/Common/DbConnectionStringCommon.cs

```diff
@@ -0,0 +1,258 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Globalization;
+
+namespace System.Data.Common
+{
+    internal static class DbConnectionStringDefaults
```

Comment:
@tinchou I suggest you dont move to common as part of this PR. This PR should only make ODBC build. 

Let's move to Common once we figure out exactly what should be moved. 

Once the compilation starts succeeding, you should clean up the files, which should be in Common folder, to remove any properties/functions which are not needed by ODBC. Right now I see a lot of them. 

Once the clean up is done and you have code which is needed only for ODBC, we can move the common properties and classes to Common folder.